Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.41326&attempt_id=2ce58768-1482-4aa8-8662-a60d5e5de1de)

Attempted: procps-ng

<details><summary>Partial log (click to expand)</summary><p>

```
   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       __bos (__s), __fmt, __va_arg_pack ());
       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
make[2]: Leaving directory '/build/procps-ng-3.3.12'
make[1]: *** [Makefile:1672: all-recursive] Error 1
make[1]: Leaving directory '/build/procps-ng-3.3.12'
make: *** [Makefile:950: all] Error 2
builder for '/nix/store/1m927wg2l6rniks5lhjjdszzlc944wk6-procps-3.3.12.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/1m927wg2l6rniks5lhjjdszzlc944wk6-procps-3.3.12.drv' failed
```
</p></details>

--- a/pkgs/os-specific/linux/procps-ng/default.nix
+++ b/pkgs/os-specific/linux/procps-ng/default.nix
@@ -1,15 +1,17 @@
-{ lib, stdenv, fetchurl, ncurses }:
+{ lib, stdenv, fetchurl, ncurses, pkgconfig }:
 
 stdenv.mkDerivation rec {
   name = "procps-${version}";
-  version = "3.3.12";
+  version = "3.3.15";
 
+  # The project's releases are on SF, but git repo on gitlab.
   src = fetchurl {
     url = "mirror://sourceforge/procps-ng/procps-ng-${version}.tar.xz";
-    sha256 = "1m57w6jmry84njd5sgk5afycbglql0al80grx027kwqqcfw5mmkf";
+    sha256 = "0r84kwa5fl0sjdashcn4vh7hgfm7ahdcysig3mcjvpmkzi7p9g8h";
   };
 
   buildInputs = [ ncurses ];
+  nativeBuildInputs = [ pkgconfig ];
 
   makeFlags = "usrbin_execdir=$(out)/bin";
 
@@ -22,7 +24,7 @@ stdenv.mkDerivation rec {
       "ac_cv_func_realloc_0_nonnull=yes" ];
 
   meta = {
-    homepage = https://sourceforge.net/projects/procps-ng/;
+    homepage = https://gitlab.com/procps-ng/procps;
     description = "Utilities that give information about processes using the /proc filesystem";
     priority = 10; # less than coreutils, which also provides "kill" and "uptime"
     license = lib.licenses.gpl2;

